userguide/platform/userguide_configuration.adoc

```diff
@@ -103,283 +104,142 @@ This sets `org.killbill.invoice.dryRunNotificationSchedule=48h` for the `boblaza
-|Persistent Bus   |org.killbill.persistent.bus.${instanceName}.queue.mode + 
-
-(Replace `<instancename>` with the persistent bus instance name (typically, `main` or `external` as specified <<persistent_bus,above>>)  |How entries are put in the queue. Possible values are  `STICKY_EVENTS`, `STICKY_POLLING` ,`POLLING` as specified in https://github.com/killbill/killbill-commons/blob/4b8d8e647b8028076b5e74cdc27170bda50ca029/queue/src/main/java/org/killbill/queue/api/PersistentQueueConfig.java[PersistentQueueMode]    |STICKY_EVENTS   
-|Persistent Bus   |org.killbill.persistent.bus.${instanceName}.nbThreads  + 
-
-(Replace `<instancename>` with the persistent bus instance name (typically, `main` or `external` as specified <<persistent_bus,above>>)  |Max number of dispatch threads to use |30   
-|Persistent Bus   |org.killbill.persistent.bus.${instanceName}.claimed + 
-
-(Replace `<instancename>` with the persistent bus instance name (typically, `main` or `external` as specified <<persistent_bus,above>>)   |Number of bus events to fetch from the database at once (only valid in 'polling mode')   |10   
-|Persistent Bus   |org.killbill.persistent.bus.${instanceName}.queue.capacity + 
-
-(Replace `<instancename>` with the persistent bus instance name (typically, `main` or `external` as specified <<persistent_bus,above>>)   |Size of the inflight queue (only valid in STICKY_EVENTS mode)   |30000   
-|Kill Bill Web Application   |org.killbill.server.multitenant   |Whether multi-tenancy is enabled   |true   
-|Kill Bill Web Application   |org.killbill.server.test.mode   |Whether to start in test mode   |false   
-|Kill Bill Web Application    |org.killbill.server.http.gzip   |Allow Kill Bill to return gzip json when Content-Encoding is set with gzip   |false   
-|Payment   |org.killbill.payment.plugin.timeout   |Timeout for each payment attempt   |30s   
-|===
+Some of the important global properties are listed in the <<Configuration Properties Table>> below.
 
 For a complete list of all the global properties, you can check the <<config_classes, config classes>> specified above.
 
-
+[[system_properties]]
 === Java System Properties 
 
-It is also possible to specify the Kill Bill configuration properties as Java system properties. For example, the `org.killbill.dao.url` can be specified as a system property via `-Dorg.killbill.dao.url=<url>`.
-
-*Important Notes:*
-
-* System properties are typically used to specify the location of third party configuration files.
-* They should generally not be used to specify regular properties. 
-* If a `killbill.properties` file is specified, the system properties are ignored and the properties specified in the properties file are used. 
+Some of the Kill Bill configuration properties need to be specified as Java system properties. 
```

Comment:
Quick question, is there any specific reason for using system properties and not setting these properties like other regular configuration properties?